hardware/targets.json

```diff
@@ -340,8 +340,8 @@
         "vrx": {
             "esp32": {
                 "product_name": "Built-in ESP32 Backpack",
-                "firmware": "HDZero_RX51_ESP32_Backpack",
-                "upload_methods": ["wifi"],
+                "firmware": "HDZero_Goggle_ESP32_Backpack",
+                "upload_methods": ["wifi", "zip"],
```

Comment:
no changes on the flash/build side to accommodate new flashing method? or is it implemented already?

Reply:
I need to implement it in the flasher. Main point is the wrong firmware was being selected at flash time!